package-lock.json

```diff
@@ -0,0 +1,8466 @@
+{
```

Comment:
Delete package-lock.json. It will conflict with yarn lock

Reply:
agreed